packages/sveltekit/src/hooks.server.ts

```diff
@@ -0,0 +1,16 @@
+import type {Handle} from "@sveltejs/kit";
+
+export const RequiredWhitelistedHeaders: ReadonlyArray<string> = [
+    'content-range'
+]
```

Comment:
Technically this could be done by just having a direct string here, but I'm not sure if Supabase has other headers, so to avoid breaking people again in case this needs to change, I thought I might as well keep it as an array right away.